README.adoc

```diff
@@ -13,14 +13,20 @@ image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 :note-caption: :information_source:
 NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
 
-It is very easy to analyze a C, C++ and Objective-C project with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
+It is very easy to analyze a C, C++ and Objective-C project with SonarCloud with coverage and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
 
 . Create a `sonar-project.properties` file to store your configuration
 . In your `.github/workflows/build.yml` file:
 .. Download the https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/[Sonar Scanner]
 .. Download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
 .. Wrap your compilation with the Build Wrapper
-.. Run `sonar-scanner` as the final step
+* Add the `-fprofile-instr-generate -fcoverage-mapping` compilation flags to instrument the binaries
+.. Run the tests to generate the test coverage information
+* After each test, "default.profraw" file will be generated. Make sure to move it someplace, to avoid overwriting with another test run.
+.. Merge the coverage information from all test runs and convert it into a format compatible with llvm-cov using `llvm-profdata merge`
+.. Run `llvm-cov show` and feed it all the test files (or all object files you are interested in).
```

Comment:
```suggestion
.. Run `llvm-cov show` and feed it with all the test files (or all object files) you are interested in.
```

Reply:
This is not what I meant to say in this sentence. Would the following be more understandable, @amelie-renard-sonarsource ?
```suggestion
.. Run `llvm-cov show` on all the test executable files (or all object files you are interested in).
```

Reply:
Yes, that's better, thanks!